security/symmetric/readme.md

```diff
@@ -0,0 +1,6 @@
+Security module used to provide symmetric key authentication capabilities to the Azure IoT Hub device client and Azure IoT Hub Provisioning Service device client.
```

Comment:
>Security [](start = 0, length = 8)

at least a link to the docs and what it's for? that isn't nearly enough if customers land on this package page on NPM

Reply:
Took the beautiful example of x509.

---
In reply to: [231364857](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231364857) [](ancestors = 231364857)